lib/ruby_lsp/internal.rb

```diff
@@ -10,6 +10,7 @@
 require "cgi"
 
 require "ruby-lsp"
+require "ruby_lsp/requests/support/uri"
```

Comment:
Since it's a monkey patch, I prefer putting it under somewhere like `lib/core_ext/uri.rb` to better express this fact.